scripts/video.js

```diff
@@ -154,6 +155,29 @@ H5P.Video = (function ($, ContentCopyrights, MediaCopyright, handlers) {
       return handlerName;
     };
 
+    /**
+    * @public
+    * Get current state for resetting it later.
+    *
+    * @returns {object} Current state.
+    */
+    self.getCurrentState = function () {
+      return {
+        startAt: self.getCurrentTime() || parameters.startAt
+      }
+    };
+
+    /**
+    * @public
+    * Reset task.
```

Comment:
I know these kinds of comments exist in old code, but this one adds no information and is strictly speaking a violation of DRY. The function is literally named resetTask already.